other/js/websockify.js

```diff
@@ -161,7 +162,23 @@ if (argv.cert) {
     console.log("    - Running in unencrypted HTTP (ws://) mode");
     webServer = http.createServer(http_request);
 }
+
+if (argv["auth-plugin"]) {
+    let auth_plugin_arg = argv["auth-plugin"].split(".")
+    let plugin_name = auth_plugin_arg.pop();
+    let module_path = auth_plugin_arg.join(".");
+
+    let auth_plugin = require(module_path)[plugin_name];
```

Comment:
In node it's not unusual for the module itself to have a single export (exports = myFunc). That case should probably be supported.

Reply:
Addressed in https://github.com/novnc/websockify/pull/285/commits/ff1897ee94b2b15bc5cff357e22d36e99f9f1164